Gemfile

```diff
@@ -9,7 +9,6 @@ gem 'amazing_print'
 gem 'pry-byebug'
 gem 'rake', '~> 13.0'
 gem 'rubocop'
-gem 'shaped', github: 'davidrunger/shaped'
```

Comment:
(`shaped` is still listed in the gemspec. we were previously requiring it here, too, because the gemspec does not support sourcing from github. now that we can source `shaped` from rubygems, we no longer need to list it here as well.)